### DISCUSSION TITLE
Fix SVGA Render and SVGA to PCem v17?

### DISCUSSION THREAD

**Discussion by wprball** [PCBox/PCBox#36] [Q&A] [answered]
Hello. Just saw these nVidia RIVA cards before, and then used them. So I installed Windows XP before, but I see SVGA Render going on here on nVidia RIVA cards cause of glitching, and not working. So, Can't you fix SVGA Render and SVGA to PCem v17?

**Comment by bachimusprime** [comment, marked as answer]
Yes, the nVidia cards don't work. This is obvious because they are only enabled in development builds. I've also noted this elsewhere on the repository. This is a more experimental program, so expecting experimental features to work is stupid. Also, PCem v17 doesn't have any nVidia cards, but thanks for playing.

EDIT: Wow, I just realized that you literally already asked this question and got the same exact answer. Please read before asking again.
